src/test/groovy/org/gradle/android/writer/Kotlin.groovy

```diff
@@ -0,0 +1,13 @@
+package org.gradle.android.writer
+
+class Kotlin {
```

Comment:
I dislike the name `Kotlin` as a class name. I'm thinking it might be better to group these code writing classes to something like `CodeSnippets`. Then we can call `CodeSnippets.getKotlinDataClass()` and `CodeSnippets.getJavaClass()`. WDYT?

Reply:
sounds good. 
Grouped all sources/resources in`CodeSnippets` https://github.com/gradle/android-cache-fix-gradle-plugin/pull/477/commits/c101f757302e52ceac73366942d47fbfb7fe0ed6